src/coreclr/src/vm/gcenv.h

```diff
@@ -44,7 +44,7 @@
 #include "gcenv.interlocked.h"
 #include "gcenv.interlocked.inl"
 
-#ifdef TARGET_UNIX
```

Comment:
Why is this HOST? Would it be better to exclude it from the DAC build?

Reply:
It was only OS page size defined here.  It is an OS feature, but perhaps the DAC doesn't really need it.